AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/testRunner/TestMatrixStore.kt

```diff
@@ -389,5 +375,5 @@ internal class TestMatrixStore(
         )
     }
 
-    private fun TestRun.Id.resultGcsPath() = (resultsGcsPrefix + key.name)
+    private fun TestRun.Id.resultGcsPath() = (resultsGcsPrefix + key.name + UUID.randomUUID().toString())
```

Comment:
sorry i wasn't very clear in the first review but this is a bit misleading.

e.g. someonce can do:

foo(testRunKey.resultGcsPath())
bar(testRunKey.resultGcsPath())

which will end up calling foo and bar with different ids.

we should rename this method ot "createGcsPath(testRun)", and localize it when needed.

Reply:
Renamed the method now. Sorry I misread your previous comment.